src/features/crops/components/Soil.tsx

```diff
@@ -73,6 +75,9 @@ export const Soil: React.FC<Props> = ({
   if (timeLeft > 0) {
     const percentage = 100 - (timeLeft / crop.harvestSeconds) * 100;
     const isAlmostReady = percentage >= 50;
+    if (timeLeft < 2) {
```

Comment:
Also here, the sound would play twice as it goes to 1 and 0..

Reply:
I think what might work best is a sound plays only on the first crop done, then every X seconds after that. Bonus points for X being user configured. Triple bonus points for user configurable ready sounds. (Chooser for in game sounds, or, user"upload" to local cache?)

Reply:
Yeah, good idea for it playing on the first crop ready.

We just need to take into considerations players can plant different crops at different times.
Having crops ready with a couple minutes in between.

I think the time being configure by player could be done based on the player feedback for this first implementation.

And the uploading sounds I don't think is ideal, this would be unnecessary complexity and also we want the sfx to align with the other sfx in game.

Reply:
> Also here, the sound would play twice as it goes to 1 and 0..
I originally had "< 1" and it never fired.  With "< 2" it fired reliably and sounded good to me.

Reply:
> I think what might work best is a sound plays only on the first crop done

I don't know what this means when crops arrive 1 sec, 5 sec, 10 sec, and 5 minutes apart.